lucene/core/src/test/org/apache/lucene/util/hnsw/KnnGraphTester.java

```diff
@@ -578,6 +572,8 @@ private int createIndex(Path docsPath, Path indexPath) throws IOException {
     IndexWriterConfig iwc = new IndexWriterConfig().setOpenMode(IndexWriterConfig.OpenMode.CREATE);
     // iwc.setMergePolicy(NoMergePolicy.INSTANCE);
     iwc.setRAMBufferSizeMB(1994d);
+    iwc.setMaxBufferedDocs(10000);
```

Comment:
Oh I did not mean to include this change here. Probably we will want to have some command line parameter to control this, but for now having the default be to index a large segment is probably better